project/Alkalid/README.md

```diff
@@ -0,0 +1,26 @@
+# ULTRA LOW BIT: The Multiple One-Bit Quantization
+
+## Basic Information
+Artificial Intelligence (AI) applications have become widespread, and in recent years, they have been extensively applied on edge devices. However, the computational demands of deep learning models are often too great for such devices to handle. To address this, many applications use a method called quantization, which reduces model parameters to 8 bits (integer) or even fewer, such as one bit. One of the biggest advantages of one-bit quantization is that binary multiplication can be simplified to an XNOR operation, which is more efficient in SIMD computations.
+
+## Problem to Solve
+Running deep learning models on edge devices with one-bit quantization greatly benefits computational efficiency but also reduces the model's precision significantly, which in turn, can lead to a substantial decrease in accuracy. Therefore, I aim to implement a "multiple one-bit" algorithm to improve precision while still maintaining the advantages of XNOR instructions.
```

Comment:
What is the value range before the quantification?

Reply:
No problem, I added it